commerce_coordinator/apps/commercetools/views.py

```diff
@@ -93,21 +98,47 @@ def post(self, request):
         canvas_entry_properties.update(extract_ct_order_information_for_braze_canvas(customer, order))
 
         for item in get_edx_items(order):
-            logger.debug('[CT-OrderFulfillView] processing edX order %s, line item %s', order_id, item.variant.sku)
+            logger.debug('[CT-OrderFulfillView] processing edX order %s, line item %s', order.id, item.variant.sku)
+
+            state_ids = []
+            for item_state in item.state:
```

Comment:
Based on the outcome of [this discussion](https://github.com/frontastic-developers/customer-twou/pull/98/files#r1539093936), we may not need to iterate through all the line item states and instead would want to only update the first state in the array. This may change in an upcoming commit.